### PR TITLE
Update industries to match GICS

### PR DESCRIPTION
**Pull request by doomspork**
Updating all of the industries to match the 11 outlined by the GISC and currently in the industries.ex file.

https://www.msci.com/our-solutions/indexes/gics

### CODE REVIEW

File: priv/companies/diatom_enterprise_softwares.exs [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
+  industry: "Technology",
   website: "http://www.diatomenterprises.com/",
   github: "https://github.com/DiatomEnterprises",
   location: %{
     city: "",
     state: "",
     country: ""
   },
-  name: "Diatom Enterprise Softwares",
+  name: "Diatom Technologys",
```
Comment (doomspork) [inline priv/companies/diatom_enterprise_softwares.exs]:
```suggestion
  name: "Diatom Enterprise Softwares",
```

File: priv/companies/diatom_enterprise_softwares.exs [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
   last_changed_on: ~D[2023-03-01],
   description: """
-  Diatom Enterprise Softwares - Latvia based custom outsource software development company producing software for a wide variety of industries. We are proficient in Elixir, .NET, Java, Ruby on Rails, Node.JS, Angular.JS and other technologies. We offer Web, Mobile and Desktop development.
+  Diatom Technologys - Latvia based custom outsource software development company producing software for a wide variety of industries. We are proficient in Elixir, .NET, Java, Ruby on Rails, Node.JS, Angular.JS and other technologies. We offer Web, Mobile and Desktop development.
```
Comment (doomspork) [inline priv/companies/diatom_enterprise_softwares.exs:13]:
```suggestion
  Diatom Enterprise Softwares - Latvia based custom outsource software development company producing software for a wide variety of industries. We are proficient in Elixir, .NET, Java, Ruby on Rails, Node.JS, Angular.JS and other technologies. We offer Web, Mobile and Desktop development.
```

File: priv/companies/enterprise_software_modules.exs [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
+  industry: "Technology",
   website: "https://www.enterprisemodules.com",
   github: "https://github.com/enterprisemodules",
   location: %{
     city: "",
     state: "",
     country: ""
   },
-  name: "Enterprise Software Modules",
+  name: "Technology Modules",
```
Comment (doomspork) [inline priv/companies/enterprise_software_modules.exs:10]:
```suggestion
  name: "Enterprise Software Modules",
```

File: priv/companies/enterprise_software_modules.exs [hunk 4]
```diff
@@ -1,15 +1,15 @@
...
   last_changed_on: ~D[2023-03-01],
   description: """
-  Supported Enterprise Software Puppet Modules for complex software. Elixir used in production for modules subscription management. Nieuwegein, Netherlands.
+  Supported Technology Puppet Modules for complex software. Elixir used in production for modules subscription management. Nieuwegein, Netherlands.
```
Comment (doomspork) [inline priv/companies/enterprise_software_modules.exs:13]:
```suggestion
  Supported Enterprise Software Puppet Modules for complex software. Elixir used in production for modules subscription management. Nieuwegein, Netherlands.
```

File: priv/companies/icicle_technologies.exs [hunk 5]
```diff
@@ -10,6 +10,6 @@
   name: "Icicle Technologies",
   last_changed_on: ~D[2023-03-01],
   description: """
-  Provides Technology Consulting, Product Development on Elixir & Phoenix. Organizer of Bangalore Elixir Group, built [ElixirNation.io](http://elixirnation.io).
+  Provides Technology, Product Development on Elixir & Phoenix. Organizer of Bangalore Elixir Group, built [ElixirNation.io](http://elixirnation.io).
```
Comment (doomspork) [inline priv/companies/icicle_technologies.exs:13]:
```suggestion
  Provides Technology Consulting, Product Development on Elixir & Phoenix. Organizer of Bangalore Elixir Group, built [ElixirNation.io](http://elixirnation.io).
```

File: priv/companies/stride_technology_consulting.exs [hunk 6]
```diff
@@ -1,13 +1,13 @@
...
+  industry: "Technology",
   website: "http://www.stridenyc.com/",
   github: "https://github.com/stride-nyc",
   location: %{
     city: "",
     state: "",
     country: ""
   },
-  name: "Stride Technology Consulting",
+  name: "Stride Technology",
```
Comment (doomspork) [inline priv/companies/stride_technology_consulting.exs:10]:
```suggestion
  name: "Stride Technology Consulting",
```

**Comment by doomspork** [comment]
@samuelpordeus @gemantzu I'm actually thinking what might be better here is further alignment with GISC to include sector, group, and sector: https://www.msci.com/our-solutions/indexes/gics

Thoughts? Maybe I'm over complicating this 🤔

**Comment by doomspork** [comment]
Hello @dzachrich and @LatitudeAdjustment! Appreciate you reaching out on LI @dzachrich to get involved with the Elixir Companies project 🎉 

One of the first areas I'm looking to fix up (it's been an ongoing issue) is our use of Industries. This PR is just a spike and I'm not admittedly in love with this so would welcome your feedback here as a kicking off point!

**Comment by doomspork** [comment]
Using GICS Sector + Industry, this is the complete list: 

ENERGY > ENERGY EQUIPMENT & SERVICES
ENERGY > OIL, GAS & CONSUMABLE FUELS

MATERIALS > CHEMICALS
MATERIALS > CONSTRUCTION MATERIALS
MATERIALS > CONTAINERS & PACKAGING
MATERIALS > METALS & MINING
MATERIALS > PAPER & FOREST PRODUCTS

INDUSTRIALS > AEROSPACE & DEFENSE
INDUSTRIALS > BUILDING PRODUCTS
INDUSTRIALS > CONSTRUCTION & ENGINEERING
INDUSTRIALS > ELECTRICAL EQUIPMENT
INDUSTRIALS > INDUSTRIAL CONGLOMERATES
INDUSTRIALS > MACHINERY
INDUSTRIALS > COMMERCIAL SERVICES & SUPPLIES
INDUSTRIALS > PROFESSIONAL SERVICES
INDUSTRIALS > AIR FREIGHT & LOGISTICS
INDUSTRIALS > AIRLINES
INDUSTRIALS > MARINE
INDUSTRIALS > ROAD & RAIL
INDUSTRIALS > TRANSPORTATION INFRASTRUCTURE
INDUSTRIALS > TRADING COMPANIES & DISTRIBUTORS

CONSUMER DISCRETIONARY > AUTO COMPONENTS
CONSUMER DISCRETIONARY > AUTOMOBILES
CONSUMER DISCRETIONARY > HOUSEHOLD DURABLES
CONSUMER DISCRETIONARY > LEISURE PRODUCTS
CONSUMER DISCRETIONARY > TEXTILES, APPAREL & LUXURY GOODS
CONSUMER DISCRETIONARY > HOTELS, RESTAURANTS & LEISURE
CONSUMER DISCRETIONARY > DIVERSIFIED CONSUMER SERVICES
CONSUMER DISCRETIONARY > DISTRIBUTORS
CONSUMER DISCRETIONARY > INTERNET & DIRECT MARKETING RETAIL
CONSUMER DISCRETIONARY > MULTILINE RETAIL
CONSUMER DISCRETIONARY > SPECIALTY RETAIL

CONSUMER STAPLES > FOOD & STAPLES RETAILING
CONSUMER STAPLES > BEVERAGES
CONSUMER STAPLES > FOOD PRODUCTS
CONSUMER STAPLES > TOBACCO
CONSUMER STAPLES > HOUSEHOLD PRODUCTS
CONSUMER STAPLES > PERSONAL PRODUCTS

HEALTH CARE > HEALTH CARE EQUIPMENT & SUPPLIES
HEALTH CARE > HEALTH CARE PROVIDERS & SERVICES
HEALTH CARE > HEALTH CARE TECHNOLOGY
HEALTH CARE > BIOTECHNOLOGY
HEALTH CARE > PHARMACEUTICALS
HEALTH CARE > LIFE SCIENCES TOOLS & SERVICES

FINANCIALS > BANKS
FINANCIALS > THRIFTS & MORTGAGE FINANCE
FINANCIALS > DIVERSIFIED FINANCIAL SERVICES
FINANCIALS > CONSUMER FINANCE
FINANCIALS > CAPITAL MARKETS
FINANCIALS > MORTGAGE REAL ESTATE INVESTMENT TRUSTS (REITS)
FINANCIALS > INSURANCE

INFORMATION TECHNOLOGY > IT SERVICES
INFORMATION TECHNOLOGY > SOFTWARE
INFORMATION TECHNOLOGY > COMMUNICATIONS EQUIPMENT
INFORMATION TECHNOLOGY > TECHNOLOGY HARDWARE, STORAGE & PERIPHERALS
INFORMATION TECHNOLOGY > ELECTRONIC EQUIPMENT, INSTRUMENTS & COMPONENTS
INFORMATION TECHNOLOGY > SEMICONDUCTORS & SEMICONDUCTOR EQUIPMENT

COMMUNICATION SERVICES > DIVERSIFIED TELECOMMUNICATION SERVICES
COMMUNICATION SERVICES > WIRELESS TELECOMMUNICATION SERVICES
COMMUNICATION SERVICES > MEDIA
COMMUNICATION SERVICES > ENTERTAINMENT
COMMUNICATION SERVICES > INTERACTIVE MEDIA & SERVICES

UTILITIES > ELECTRIC UTILITIES
UTILITIES > GAS UTILITIES
UTILITIES > MULTI-UTILITIES
UTILITIES > WATER UTILITIES
UTILITIES > INDEPENDENT POWER AND RENEWABLE ELECTRICITY PRODUCERS

REAL ESTATE > EQUITY REAL ESTATE INVESTMENT TRUSTS (REITS)
REAL ESTATE > REAL ESTATE MANAGEMENT & DEVELOPMENT

**Comment by doomspork** [comment]
With the help of ChatGPT I got the data formatted in a way we can actually use it:

```elixir
%{
  "Communication Services" => [
    "Interactive Media & Services",
    "Entertainment",
    "Media",
    "Wireless Telecommunication Services",
    "Diversified Telecommunication Services"
  ],
  "Consumer Discretionary" => [
    "Specialty Retail",
    "Multiline Retail",
    "Internet & Direct Marketing Retail",
    "Distributors",
    "Diversified Consumer Services",
    "Hotels, Restaurants & Leisure",
    "Textiles, Apparel & Luxury Goods",
    "Leisure Products",
    "Household Durables",
    "Automobiles",
    "Auto Components"
  ],
  "Consumer Staples" => [
    "Personal Products",
    "Household Products",
    "Tobacco",
    "Food Products",
    "Beverages",
    "Food & Staples Retailing"
  ],
  "Energy" => ["Oil, Gas & Consumable Fuels", "Energy Equipment & Services"],
  "Financials" => [
    "Insurance",
    "Mortgage Real Estate Investment Trusts (reits)",
    "Capital Markets",
    "Consumer Finance",
    "Diversified Financial Services",
    "Thrifts & Mortgage Finance",
    "Banks"
  ],
  "Health Care" => [
    "Life Sciences Tools & Services",
    "Pharmaceuticals",
    "Biotechnology",
    "Health Care Technology",
    "Health Care Providers & Services",
    "Health Care Equipment & Supplies"
  ],
  "Industrials" => [
    "Trading Companies & Distributors",
    "Transportation Infrastructure",
    "Road & Rail",
    "Marine",
    "Airlines",
    "Air Freight & Logistics",
    "Professional Services",
    "Commercial Services & Supplies",
    "Machinery",
    "Industrial Conglomerates",
    "Electrical Equipment",
    "Construction & Engineering",
    "Building Products",
    "Aerospace & Defense"
  ],
  "Information Technology" => [
    "Semiconductors & Semiconductor Equipment",
    "Electronic Equipment, Instruments & Components",
    "Technology Hardware, Storage & Peripherals",
    "Communications Equipment",
    "Software",
    "It Services"
  ],
  "Materials" => [
    "Paper & Forest Products",
    "Metals & Mining",
    "Containers & Packaging",
    "Construction Materials",
    "Chemicals"
  ],
  "Real Estate" => ["Real Estate Management & Development", "Equity Real Estate Investment Trusts (reits)"],
  "Utilities" => [
    "Independent Power And Renewable Electricity Producers",
    "Water Utilities",
    "Multi-utilities",
    "Gas Utilities",
    "Electric Utilities"
  ]
}
```

File: priv/companies/ybex.exs [hunk 7]
```diff
@@ -10,6 +10,6 @@
   name: "YBEX",
   last_changed_on: ~D[2023-03-01],
   description: """
-  YBEX is a financial technology company, providing a marketplace platform for retail foreign exchange services. Elixir is used for our backend services. Based in Hong Kong.
+  YBEX is a Financials company, providing a marketplace platform for retail foreign exchange services. Elixir is used for our backend services. Based in Hong Kong.
```
Comment (tonnenpinguin) [inline priv/companies/ybex.exs:13]: This was probably not on purpose :)

Comment (doomspork) [reply priv/companies/ybex.exs:13]:
```suggestion
  YBEX is a financial technology company, providing a marketplace platform for retail foreign exchange services. Elixir is used for our backend services. Based in Hong Kong.
```

**Pull request closed without merging**